.github/workflows/release.yml

```
@@ -13,66 +13,84 @@ jobs:
-          # Verify the update
           grep -m 1 'version = ' Cargo.toml
 
       - name: Check generate lock
-        if: steps.check_pr.outputs.release_exists != 'true'
+        if: steps.check_pr.outputs.exists != 'true'
         run: cargo generate-lockfile
 
       - name: Create Pull Request
-        if: steps.check_pr.outputs.release_exists != 'true'
+        if: steps.check_pr.outputs.exists != 'true'
         uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.12)

[error] the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)